.docker/requirements.txt

```diff
@@ -18,4 +18,4 @@ pytest-xdist
 #
 
 dependency_injector == 4.41.0
-delta-spark==3.1.0
+delta-spark>=3.1,<4.0
```

Comment:
I think the requirement is better off with a specific version. I believe you can revert this change.

Reply:
done